src/inc/zapper.h

```diff
@@ -126,7 +126,11 @@ class Zapper
 
 #if defined(FEATURE_CORECLR) && !defined(FEATURE_MERGE_JIT_AND_ENGINE)
     SString                 m_CLRJITPath;
+    bool                    m_fDontLoadJit;
```

Comment:
> his bool variable is still needed to control whether or not to load JIT

If JITPath is not specified, shouldn't that suffice since the path will be null?